### DISCUSSION TITLE
missing dependency

### DISCUSSION THREAD

**Discussion by keremcozin** [mhx/dwarfs#140] [General]
Hello first of all, thanks for this project.

I'm a Gentoo GNU/Linux user, I pulled and compiled 'dwarfs' package from Guru overlay. In that ebuild  'dev-libs/fsst' dependency is missing but it's needed by dwarfs .

**Comment by mhx** [comment]
Hi @keremcozin!

I'm a Gentoo user as well, but unfortunately have nothing to do with the ebuilds in the Guru overlay. I'm not aware that `fsst` even exists as a library, this is probably work in progress. I know that @RarogCmex worked on the ebuilds in the past, but I don't know what the state is.

In the meantime, it's really your best bet to install the binaries from the release page.

**Reply by keremcozin** [reply]
Hi again, thanks for the reply.
It's all good, source package runs as intended without any issues.

**Comment by RarogCmex** [comment]
@keremcozin There are also dwarfs-bin which simply installs binary dwarfs files.
Thank you, I will try to fix it when I have time.
P.S. Sorry, I didn't noticed github message.

**Comment by RarogCmex** [comment]
Hopefully, fsst is present in Gentoo tree (and already installed in my system) so I can fix it within 5 minutes. However, be warned that after my change there will be gentoo guru code review which usually takes week or two.

**Comment by RarogCmex** [comment]
To be honest, dwarfs ebuilds was been even deleted once because I didn't have much time. Maybe it had an issue for a half of year and no one noticed :)
